client/src/components/RoundAddDialog.vue

```diff
@@ -88,6 +89,10 @@ export default {
       type: String,
       required: true,
     },
+    gameTeam: {
+      type: Object,
+      required: true
+    }
```

Comment:
Instead of passing team as a prop I would rather use `team` getter if it has coop information already (I believe it does). Team getter queries team by id from the database

Reply:
ok on it